types/params_test.go

```diff
@@ -449,6 +488,53 @@ func TestConsensusParamsUpdate_VoteExtensionsEnableHeight(t *testing.T) {
 	}
 }
 
+func TestConsensusParamsUpdate_PBTSEnableHeight(t *testing.T) {
```

Comment:
Can you introduce an exhaustive UT (like `TestConsensusParamsUpdate_VoteExtensionsEnableHeight` above)?
Or maybe you can reuse `TestConsensusParamsUpdate_VoteExtensionsEnableHeight` to also test PBTS enable height, as the logic should be the same

Reply:
Done in [`TestConsensusParamsUpdate_EnableHeight`](https://github.com/cometbft/cometbft/blob/bd464ea7f8870107e15598fadc5d208d134da330/types/params_test.go#L467-L468)